### PR TITLE
[Ember-Migration] Footer Enhancement

### LINKED ISSUE #722: [Ember-Migration] Footer Enhancement
### Description:
The current footer in live(beta) site under `dev=true` feature flag looks like this:

Screenshot_7

However this footer should be different for other pages except homepage. The marked potion in the below screenshot should not be displayed on other pages and only `The contents of this website are deployed from this open sourced repo` part should be displayed on other pages.

Screenshot_8

### Notes

There will be no design change from current footer, only contents will be different.

### PR DESCRIPTION
**Pull request by ajoykumardas12**
closes #722 

### Issue
- #722 

### What is the change?
- This change dynamically renders footer depending on route. Only `The contents of this website are deployed from this open sourced repo` part is visible in all pages and in home route events and other parts of footer is visible.

### Frontend Changes
- [x] Yes
- [ ] No

### Backend Changes
- [ ] Yes
- [x] No

### Feature Flag
- [x] Yes
- [ ] No

### Is Development Tested?

- [x] Yes
- [ ] No

https://github.com/Real-Dev-Squad/website-www/assets/87164140/388022d0-a8dd-4051-8daf-9c233a15357d

### Visual of changes

https://github.com/Real-Dev-Squad/website-www/assets/87164140/e1a08462-c727-41e2-a013-4b23d5df3306

### CODE REVIEW

File: app/components/footer.hbs
Comment (satyam73) [inline app/components/footer.hbs:1]: any tests for the newly added part?

**Review by satyam73 — CHANGES_REQUESTED** [review]

File: app/components/footer.hbs
Comment (ajoykumardas12) [reply app/components/footer.hbs:1]: I have added acceptance test for the changes.

**Comment by ajoykumardas12** [comment]
### Test stats for newly added tests

Screenshot_test

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by ardourApeX — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**